CMakeLists.txt

```diff
@@ -85,8 +85,9 @@ else()
 	find_package(IrrlichtMt QUIET)
 	if(NOT TARGET IrrlichtMt::IrrlichtMt)
 		string(CONCAT explanation_msg
-			"The Minetest team has forked Irrlicht to make their own customizations. "
-			"It can be found here: https://github.com/minetest/irrlicht")
+			"The Minetest team has forked Irrlicht to make their own customizations.\n"
+			"It can be found here: https://github.com/minetest/irrlicht\n"
+			"For example: git clone --depth=1 https://github.com/minetest/irrlicht lib/irrlichtmt\n")
```

Comment:
```suggestion
			"The Minetest team has forked Irrlicht to make their own customizations. "
			"It can be found here: https://github.com/minetest/irrlicht\n"
			"For example use: git clone --depth=1 https://github.com/minetest/irrlicht lib/irrlichtmt\n")
```